swift-sdk/IterableAPI.swift

```diff
@@ -431,6 +431,21 @@ public final class IterableAPI: NSObject {
         internalImplementation?.trackInAppClick(messageId, saveToInbox: saveToInbox, silentInbox: silentInbox, location: location, clickedUrl: clickedUrl)
     }
     
+    /**
+     Tracks an InAppDelivery event
+     */
+    @objc(trackInAppDelivery:) public static func track(inAppDelivery messageId: String) {
```

Comment:
Since the SDK is managing the local queue, I think we don't have to make `trackInAppDelivery` public.